css/style.css

```diff
@@ -257,6 +257,70 @@ hr {
 }
 
 
+
+******************Section-Sharpen your skills*****************/
+ table {
+  border-collapse: collapse;
+  border-spacing: 0;
+  width: 90%;
+  border: 1px solid #ddd;
+  align: centre;
+}
+
+table.center {
+  margin-left: auto; 
+  margin-right: auto;
+}
+
+
+th:hover {
+background-color: #FFF;
+}
+
+th, td {
+  text-align: left;
+  padding: 30px;
+  font-weight: bold;
+  font-size:20px;
+  text-shadow: 2px 2px #0000;
+}
+
+a:Link{
+   color:#000000;
+
+}
+
+a:visited {
+  color:#000000;
+}
+
+a:Hover{
+   color:#000000;
+}
+
+tr:nth-child(odd) {
+  background-color: #3caea3;
+}
+
+hr {
+    border: 0.5px solid #3caea3;
+    display: block;
+    width: 80px;
+    margin: auto;
+}
+
+h1 {
+    font-size: 50px;
+    font-weight: bold;
+    backdrop-filter: blur(20px);
+}
+
+p1{
```

Comment:
p1 is an invalid tag.

Reply:
because of p1 tag use there the section added is separated from footer otherwise it is touching footer.

Reply:
p1 is not a standard tag use span with some class and apply the styles to it.